src/services/API/user.js

```diff
@@ -1,9 +1,8 @@
-import { client } from "@/services/API/rootClient.js";
+import { client, newClient } from "@/services/API/rootClient.js";
```

Comment:
`newClient` is a very bad name. Call it `AFDBClient` maybe? Or something descriptive